org.eclipse.jdt.debug.ui/ui/org/eclipse/jdt/internal/debug/ui/console/JavaStackTraceHyperlink.java

```diff
@@ -203,6 +252,96 @@ public IStatus runInUIThread(IProgressMonitor monitor) {
 		job.schedule();
 	}
 
+	/**
+	 * Filters results based on method signature
+	 *
+	 * @param type
+	 *            Type of the matches
+	 * @param methodSignature
+	 *            entire method declaration
+	 * @param methodNameExtracted
+	 *            method name
+	 * @throws JavaModelException
+	 */
+	private boolean extractFromResults(IType type, String methodSignature, String methodNameExtracted) throws JavaModelException {
```

Comment:
What does the method actually? 
return `true` means **what exactly**? Can we please explain that in javadoc

Reply:
Sure @iloveeclipse , will include javadoc for every new methods